### PR TITLE
Add ability to look up a user's key

### PR DESCRIPTION
**Pull request by atayarani**
@marcuswalser @ehealy

### CODE REVIEW

File: bin/ldaptools [hunk 1]
```diff
@@ -23,9 +23,10 @@ when 'group'
   commands('This object is used for group management', cmd, GROUP_SUB_COMMANDS)
   Tapjoy::LDAP::Group.new
 when 'key'
-  KEY_SUB_COMMANDS = %w(add remove install)
+  KEY_SUB_COMMANDS = %w(add remove install list)
```
Comment (marcuswalser) [inline bin/ldaptools]: Is the intent here that you run ldaptools list <username> and get a key back? If so I think 'show' is better. List implies to me that you're getting all of something.

**Comment by marcuswalser** [comment]
LGTM apart from my diction nitpick.

**Comment by atayarani** [comment]
@marcuswalser why not both?

**Comment by marcuswalser** [comment]
LGTM

**Pull request merged**